### DISCUSSION TITLE
stucked at electrs-librocks-sys(build)

### DISCUSSION THREAD

**Discussion by Hor4ti0** [raspiblitz/raspiblitz#4390] [Q&A]
![Screenshot 2024-01-31 193235](https://github.com/raspiblitz/raspiblitz/assets/85168876/edb481f8-c433-436c-893c-051184af6f0c)

Seems I'm stucked at this. Tried multiple reboots, waited for hours nothing changed.

Just wanted to install Electrum.

**Comment by rootzoll** [comment]
Which version do you run?

**Comment by Hor4ti0** [comment]
Version 1.10

**Comment by rootzoll** [comment]
Please wait for the upcomming v1.11.0 release - if that problem then still exists, please write again.

**Comment by codenoxi** [comment]
same problem on my side. tried to install it already 3 times. no success.
also Version 1.10
how can I debug this to see more information about the problem?
